app/analysis/index/controller.js

```diff
@@ -35,228 +35,202 @@ const deserializeMoment = (momentString) =>
-    }
-  ),
-
-  selectedCustomer: computed(
-    "customer",
-    "prefetchData.lastSuccessful.value.customer",
-    "store",
-    function () {
-      return this.customer && this.store.peekRecord("customer", this.customer);
-    }
-  ),
-
-  selectedProject: computed(
-    "prefetchData.lastSuccessful.value.project",
-    "project",
-    "store",
-    function () {
-      return this.project && this.store.peekRecord("project", this.project);
-    }
-  ),
-
-  selectedTask: computed(
-    "prefetchData.lastSuccessful.value.task",
-    "store",
-    "task",
-    function () {
-      return this.task && this.store.peekRecord("task", this.task);
-    }
-  ),
-
-  selectedUser: computed(
-    "prefetchData.lastSuccessful.value.user",
-    "store",
-    "user",
-    function () {
-      return this.user && this.store.peekRecord("user", this.user);
-    }
-  ),
-
-  selectedReviewer: computed(
-    "prefetchData.lastSuccessful.value.reviewer",
-    "reviewer",
-    "store",
-    function () {
-      return this.reviewer && this.store.peekRecord("user", this.reviewer);
-    }
-  ),
+  get costCenters() {
+    return this.store.findAll("cost-center");
+  }
 
-  exportLimitExceeded: computed("exportLimit", "totalItems", function () {
-    return this.totalItems > this.exportLimit;
-  }),
+  get selectedCustomer() {
+    return this.customer && this.store.peekRecord("customer", this.customer);
+  }
 
-  exportLimitMessage: computed("exportLimit", function () {
-    return `The export limit is ${this.exportLimit}. Please use filters to reduce the amount of reports.`;
-  }),
+  get selectedProject() {
+    return this.project && this.store.peekRecord("project", this.project);
+  }
 
-  canBill: computed.or(
-    "session.data.user.isAccountant",
-    "session.data.user.isSuperuser"
-  ),
+  get selectedTask() {
+    return this.task && this.store.peekRecord("task", this.task);
+  }
 
-  exportLinks: config.APP.REPORTEXPORTS,
+  get selectedUser() {
+    return this.user && this.store.peekRecord("user", this.user);
+  }
 
-  exportLimit: config.APP.EXPORT_LIMIT,
+  get selectedReviewer() {
+    return this.reviewer && this.store.peekRecord("user", this.reviewer);
+  }
 
-  session: service("session"),
+  get exportLimitExceeded() {
+    return this.totalItems > this.exportLimit;
+  }
 
-  notify: service("notify"),
+  get exportLimitMessage() {
+    return `The export limit is ${this.exportLimit}. Please use filters to reduce the amount of reports.`;
+  }
 
-  can: service("can"),
+  get canBill() {
+    return (
+      this.session.data.user.isAccountant || this.session.data.user.isSuperuser
+    );
+  }
 
-  jwt: reads("session.data.authenticated.access_token"),
+  exportLinks = config.APP.REPORTEXPORTS;
 
-  _scrollOffset: 0,
+  exportLimit = config.APP.EXPORT_LIMIT;
 
-  init(...args) {
-    this._super(...args);
+  @service session;
+  @service store;
+  @service router;
+  @service notify;
+  @service can;
```

Comment:
Nitpick, would probably move services to start of class definition since this is kind of the unspoken norm i think.